hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -217,6 +217,26 @@ static const struct usage_map xpadneo_usage_maps[] = {
 
 	/* disable duplicate button */
 	USAGE_IGN(0xC0224),
+
+	/* XBE2 hacks */
```

Comment:
Maybe put "XBE2" in front of each header, so we have that associated with each individual block instead of a semi-top-level header. It makes it easier to reorder blocks later. We should probably also add "Generic:" or "All:" to other block header comments.

It also makes it easier to follow up with some model specific logic later.